<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiviteUI</class>
 <widget class="QDialog" name="ActiviteUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/abc.jpg);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>781</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter Activité</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>441</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter Activité</string>
     </property>
     <widget class="QDateEdit" name="dateDebut">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>116</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>52</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>116</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date début :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>84</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Nom">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>52</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="type">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>84</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AjouterBouton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date fin :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateFin">
      <property name="geometry">
       <rect>
        <x>106</x>
        <y>148</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher Activité</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>381</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Supprimer Activité</string>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>17</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>12</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Activité</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SupprimerBouton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>51</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>85</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>119</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date début :</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Nomval">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>51</y>
        <width>256</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="typeval">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>85</y>
        <width>256</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Datedebutval">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>119</y>
        <width>256</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Datefinval">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>256</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date fin :</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tablemodifier">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>741</width>
       <height>251</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Modifier Activité</string>
     </property>
     <widget class="QComboBox" name="comboBox_modif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>124</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date début :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>56</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nommodif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="typemodif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>95</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Activité</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateDebutmodif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifiebtn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>154</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date fin :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateFinmodif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="recherche">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tri">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>280</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
